Makefile.am

```diff
@@ -16,7 +16,9 @@ EXTRA_DIST = \
 	build-aux/compile \
 	build-aux/depcomp \
 	build-aux/install-sh \
-	build-aux/missing
+	build-aux/missing \
+	compat/os/availability.h \
+	compat/libproc_preinclude.h
```

Comment:
These should be picket up automatically when properly used with the darwin platform.

Reply:
Got it. Removed. (I'll resolve this.)